content/en/blog/falcosidekick-openfaas.md

```diff
@@ -0,0 +1,571 @@
+$  PASSWORD=$(kubectl get secret -n openfaas basic-auth -o jsonpath="{.data.basic-auth-password}" | base64 --decode; echo)
+$echo -n $PASSWORD | faas-cli login --username admin --password-stdin
+Calling the OpenFaaS server to validate the credentials...
+credentials saved for admin http://127.0.0.1:8080
+```
+
+## Install Falco + Falcosidekick
+
+Firstly, we'll create the namespace that will host both `Falco` and `Falcosidekick`:
+
+```shell
+$ kubectl create namespace falco
+```
+
+We add the `helm` repo:
+
+```shell
+$ helm repo add falcosecurity https://falcosecurity.github.io/charts
+"falcosecurity" has been added to your repositories
+$ helm repo update
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "falcosecurity" chart repository
+...Successfully got an update from the "openfaas" chart repository
+Update Complete. ⎈Happy Helming!⎈
+```
+
+In a real project, you should get the whole chart with `helm pull falcosecurity/falco --untar` and then configure
+the `values.yaml`. For this tutorial, will try to keep thing as easy as possible and set configs directly
+by `helm install` command:
+
+```shell
+$ helm upgrade --install falco falcosecurity/falco --namespace falco \
+--set falcosidekick.enabled=true \
+--set falcosidekick.config.openfaas.gatewayservice="gateway" \
+--set falcosidekick.config.openfaas.gatewaynamespace="openfaas" \
+--set falcosidekick.config.openfaas.functionname="falco-pod-delete" \
+--set falcosidekick.config.openfaas.functionnamespace="openfaas-fn"
+```
+
+You should get this output:
+
+```shell
+Release "falco" does not exist. Installing it now.
+NAME: falco
+LAST DEPLOYED: Tue Apr 13 10:49:49 2021
+NAMESPACE: falco
+STATUS: deployed
+REVISION: 1
+NOTES:
+Falco agents are spinning up on each node in your cluster. After a few
+seconds, they are going to start monitoring your containers looking for
+security issues.
+
+
+No further action should be required.
+```
+
+And you can see your new `Falco` and `Falcosidekick`pods:
+
+```shell
+$ kubectl get pods --namespace falco
+NAME                                   READY   STATUS    RESTARTS   AGE
+falco-falcosidekick-7779579477-mwsb4   1/1     Running   0          67s
+falco-falcosidekick-7779579477-n5v89   1/1     Running   0          67s
+falco-p97rw                            1/1     Running   0          67s
+```
+
+The argument `falcosidekick.enabled=true` is going to enable following arguments by default:
+
+```shell
+--set falco.jsonOutput=true \
+--set falco.httpOutput.enabled=true \
+--set falco.httpOutput.url=http://falcosidekick:2801
+```
+
+The
+arguments `--set falco.jsonOutput=true --set falco.httpOutput.enabled=true --set falco.httpOutput.url=http://falcosidekick:2801`
+are there configuring the format of events and the URL where `Falco` will send them. As `Falco` and `Falcosidekick` will
+be in the same namespace, we can directly use the name of the service (`falcosidekick`).
```

Comment:
```suggestion
are there to configure the format of events and the URL to which `Falco` will send them. As `Falco` and `Falcosidekick` will
be in the same namespace, it can directly use the name of the service (`falco-falcosidekick`) above `Falcosidekick` pods.
```

Reply:
done